evolver/app/tests/conftest.py

```diff
@@ -1,10 +1,17 @@
-import os
 import pytest
 from fastapi.testclient import TestClient
-from ..main import app  # Leave as relative for use in template: ssec-jhu/base-template.
 
+from evolver.app.main import app
+from evolver.device import EvolverConfig
+from evolver.settings import app_settings
 
-@pytest.fixture(scope="class")
-def app_client(tmp_path_factory):
```

Comment:
``tmp_path_factory`` is session scoped whilst ``tmpdir`` and ``tmp_path`` are test invocation scoped.